### PR TITLE
Set Modeling view title

### PR DESCRIPTION
**Pull request by lewfish**
This PR sets the page title according to the model package.

##### Testing
* Log in
* Create a new TR55 project, and check that the page title is 'Model My Watershed: Site Storm Model'. Hit reload and check that title is still correct.
* Click back and check that title returns to 'Model My Watershed: Geospatial Analysis'
* Create a new GWLF-E project, and check that the page title is 'Model My Watershed: Watershed Multi-Year Model'. Hit reload and check that title is still correct.

Connects #1381

### CODE REVIEW

File: src/mmw/js/src/modeling/controllers.js [hunk 1]
```diff
@@ -227,6 +231,14 @@ function finishProjectSetup(project, lock) {
     });
 }
 
+function setPageTitle() {
+    var modelPackageName = App.currentProject.get('model_package'),
+        modelPackages = settings.get('model_packages'),
+        modelPackageDisplayName = _.find(modelPackages, {name: modelPackageName}).display_name;
+
+    App.state.set('current_page_title', modelPackageDisplayName());
```
Comment (rajadain) [inline src/mmw/js/src/modeling/controllers.js]: ~~`modelPackageDisplayName()`~~ → `modelPackageDisplayName`

File: src/mmw/js/src/modeling/controllers.js [hunk 2]
```diff
@@ -323,6 +335,7 @@ function reinstateProject(number, lock) {
                 'areaOfInterest': project.get('area_of_interest'),
                 'areaOfInterestName': project.get('area_of_interest_name')
             });
+            App.state.set('current_page_title', getModelPackageDisplayName());
```
Comment (rajadain) [inline src/mmw/js/src/modeling/controllers.js]: `getModelPackageDisplayName` doesn't exist?

**Comment by lewfish** [comment]
I updated the commit to fix the mistakes you noted.

**Comment by rajadain** [comment]
When I reload a GWLF-E project, it says "Site Storm Model" until the page finishes rendering, then switches to "Watershed Multi-Year Model"

mmw-delayed-label

Consider rendering the label without waiting for the page to finish rendering. If this is not possible, I would prefer the label to be empty, rather than incorrect, in the beginning.

**Comment by rajadain** [comment]
Apart from that little niggle above, +1 tested. Good job slipping this in there without having to do sweeping changes!

**Comment by lewfish** [comment]
Good catch. I fixed it so it now says "Model My Watershed: Modeling" while loading the project.

**Comment by rajadain** [comment]
+1

**Pull request merged**